ds_judgements_public_ui/sass/includes/_confirmation.scss

```diff
@@ -0,0 +1,17 @@
+h1.confirmation-steps {
+  margin-bottom: -1.5rem;
+}
+
+.govuk-warning-text {
```

Comment:
A few  more tweaks we'll need to make here:

![image](https://github.com/nationalarchives/ds-caselaw-public-ui/assets/4279/ddeb15fc-6b14-4888-b1fc-8b375749f5ec)


 -  The horizontal margins are inconsistent, we should tidy this up.
 -  The icon  should be vertically flush with the top of the text
 -  (I *thought* the icon for the red warning box was an exclamation mark rather than an "i" but i'm going to double check that, as i think if it was so, it should just work)
 - We have two different "warning box" styles within the form (see also ".govuk-notification-banner_red". What difference are we intending to communicate here, if any? if they are being used for different purposes, i don't think that's coming across (the difference is confusing). If they're not being used for different purposes, we should settle on one and use it consistently. This is a design thing rather than a dev thing, so not a criticism of your work, but it'd be good to deal with this while we're here.